spec/score_spec.rb

```diff
@@ -1,37 +1,34 @@
 require "./lib/score.rb"
 
-# describe Word do
-# @word = Scrabblegame.new
-# end
-
-describe "#word as parameter" do
-  it "checks if the input is a string" do
-    word = "test"
-    #.to be String/Fixnum
-    expect(word).to be_an_instance_of String
+describe Scrabble::ScrabbleGame do
+  describe "#word as parameter" do
+    it "checks if the input is a string" do
+      word = "test"
+      expect(word).to be_an_instance_of String
```

Comment:
This test isn't really testing that the parameter can be accepted as a string, but rather it is just testing the string class in ruby - it is not referencing the code you've written at all